v2/pkg/reporting/exporters/markdown/markdown.go

```diff
@@ -66,6 +66,34 @@ func (exporter *Exporter) Export(event *output.ResultEvent) error {
 	dataBuilder.WriteString(description)
 	data := dataBuilder.Bytes()
 
+	// index generation
+
+	file, err := os.OpenFile(filepath.Join(exporter.directory, "index.md"), os.O_CREATE|os.O_WRONLY, 0644)
```

Comment:
Why the file index header is written over and over? Shouldn't the index creation happen in the `New(..)`?